Helpers.ps1

```diff
@@ -800,3 +802,29 @@ function Get-HttpWebResponseContent
         }
     }
 }
+
+function Repair-LogFile
+{
+<#
+    .SYNOPSIS
+        Repais the log file encoding.
+
+    .DESCRIPTION
+        Repairs the log file by encoding it to UTF8 if it is currently UTF16.
+
+    .EXAMPLE
+        Repair-LogFile
+#>
+    $path = (Get-GitHubConfiguration -Name LogPath)
+
+    $logFileBytes = Get-Content -Encoding Byte -ReadCount 2 -Path $path
```

Comment:
```suggestion
    $logFileBytes = Get-Content -Encoding Byte -TotalCount 2 -Path $path
```

What I had transposed in the issue was wrong.  If you use `ReadCount` it will read the _entire_ file...it's just specifying how many lines to pipe through at a time.  `TotalCount` on the other hand indicates how many lines to read ... in total.  Definite perf impact of using `TotalCount` instead of `ReadCount`.